recipes/beyla-golden-signals/gce/beyla-config.yaml

```diff
@@ -0,0 +1,45 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Beyla can be configured to listen to applications running on multiple ports.
+# Beyla can also be configured globally using environment variables.
+# Environment variables have priority over the properties in the config file.
+open_port: 8080,8082,8083
+trace_printer: text
+protocol: grpc
+
+otel_metrics_export:
+  endpoint: http://0.0.0.0:4317
+  # default includes all possible instrumentations "*"
+  instrumentations: ["http", "grpc"]
+  # default only includes "application"
+  features: ["application", "application_process"]
+
+otel_traces_export:
+  endpoint: http://0.0.0.0:4317
```

Comment:
if we remove this, does it disable tracing?

Reply:
Yes, based on Beyla docs, if we remove the endpoint, trace export from Beyla will be disabled - not sure if this also means that trace generation from Beyla would be disabled too.

Reply:
lets turn it off, then.

Reply:
I think then we should switch on the trace export from the Ops Agent side - since we would be disabling the generation at the Beyla level.

Reply:
I think you mean switch off trace export?  If so, sgtm